failing on windows:

```sh
error[E0432]: unresolved import `nix::libc`
  --> crates\rustpatcher\src\updater.rs:12:5
   |
12 | use nix::libc;
   |     ^^^^^^^^^ no `libc` in the root

error[E0433]: failed to resolve: could not find `libc` in `nix`
   --> crates\rustpatcher\src\updater.rs:183:28
    |
183 |             let res = nix::libc::execv(exe.as_ptr(), args.as_ptr());
    |                            ^^^^ could not find `libc` in `nix`

Some errors have detailed explanations: E0432, E0433.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `rustpatcher` (lib) due to 2 previous errors
Error: Process completed with exit code 101.
```

will ignore for now and declare win as unsupported platform for the moment.

--- a/crates/rustpatcher/Cargo.toml
+++ b/crates/rustpatcher/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "rustpatcher"
-version = "0.2.2"
+version = "0.2.1"
 edition = "2024"
 description = "distributed patching system for single binary applications"
 license = "MIT"

